src/Monitor/Monitor/Diagnostics/SetAzureRmDiagnosticSettingCommand.cs

```diff
@@ -526,6 +533,11 @@ private void SetWorkspace(DiagnosticSettingsResource properties)
             {
                 WriteDebugWithTimestamp("Setting workspace Id");
                 properties.WorkspaceId = this.WorkspaceId;
+
+                WriteDebugWithTimestamp("Setting LogAnalyticsDestinationType");
+                properties.LogAnalyticsDestinationType = this.ExportToResourceSpecific.IsPresent
+                    ? "Dedicated"
+                    : null;
```

Comment:
what to do when a dedicated type is set?

Reply:
This is the value introduced in this change. By default LogAnalyticsDstinationType is null (since it was not there in previous versions) or it is "AzureDiagnostics''. When the user want to send the data to a different place in Log Analytics, then this value is set to Dedicated. 
The cmdlet itself does not have to do anything about this value, just send it and /or show it to the user.